os/board/rtl8730e/src/component/soc/amebad2/fwlib/rom_common/ameba_gpio_rom.c

```diff
@@ -369,6 +369,9 @@ void GPIO_WriteBit(u32 GPIO_Pin, u32 Pin_State)
 	u8 pin_num;
 	u32 RegValue;
 
+#ifdef CONFIG_PLATFORM_TIZENRT_OS
```

Comment:
Could you let me know why we need this conditional?
Other OS does not need?

Reply:
Hi, other OS also needs it, just that our default SDK freertos is using a different API for enter critical, so i think it is better to differentiate it.

Reply:
Suggestion:
If the issue occurs with SMP/AMP, using this macro CONFIG_SMP to wrap is more appropriate.
CONFIG_AMP is already covered since it depends on CONFIG_SMP [=y].

Reply:
@zhongnuo-tang @ZhenBei-Sin Could you let me know final your opinion for this?

Reply:
Hi @sunghan-chang , keep it unchanged.